log4j-core/src/main/java/org/apache/logging/log4j/core/util/UuidUtil.java

```diff
@@ -50,6 +51,7 @@ public final class UuidUtil {
     private static final int HUNDRED_NANOS_PER_MILLI = 10000;
 
     private static final long LEAST = initialize(NetUtils.getMacAddress());
+    private static final long SALT = new SecureRandom().nextLong();
```

Comment:
What is the impact of this on startup time, e.g. in a VM without a physical source of entropy?

I think that `Random` might be enough, since this is not used for cryptographic purposes.